aperturedb/cli/utilities.py

```diff
@@ -0,0 +1,21 @@
+from aperturedb.Utils import create_connector, Utils
+import typer
+from enum import Enum
+
+app = typer.Typer()
+
+
+class CommandTypes(str, Enum):
+    SUMMARY = "summary"
+    REMOVE_ALL = "remove_all"
+
+
+@app.command()
+def execute(command: CommandTypes):
+    utils = Utils(create_connector())
+    available_commands = {
+        CommandTypes.SUMMARY: utils.summary,
+        CommandTypes.REMOVE_ALL: utils.remove_all_objects
```

Comment:
We should add wrapper to utils.remove_all_objects call that confirms with the user the deletion. I see myself pressing tab and enter by accident and bye bye data 🥲

Reply:
this is done.